work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -1965,6 +1971,9 @@ def _write_config(self):
         with open(self._config_file, 'w') as f:
             json.dump(opts_subset, f, indent=4)
 
+    def _specify_environment(self, env):
```

Comment:
Suggest that you add `factory.specify_package()` and `task.specify_package()` as aliases for `specify_environment`, since all of the other tooling describes the tarballs as packages rather than environments.